### ISSUE TITLE
Automate extract of Control Tower Guardrails to Github

### ISSUE THREAD

**Issue by CalvinRodo** [cds-snc/cds-aws-lz#97]
Instead of having to put the guardrails in github we should just have an automated process that does it for every OU.

Other alternative is we just wait for guardrails to be manageable through terraform just waiting on this issue to finish up: https://github.com/hashicorp/terraform-provider-aws/issues/26612

**Comment by CalvinRodo** [comment]
Other alternative is we just wait for guardrails to be manageable through terraform

**Comment by CalvinRodo** [comment]
Closing in favour of #141
